README.md

```diff
@@ -2,6 +2,17 @@
 
 REST server that indexes search documents within an Atlas cluster and handles user search requests, deployed via Kanopy
 
+## Components
+
+### Marian Server
+REST server that exposes endpoints to run queries against A DB.
```

Comment:
(Super duper nit) Should we lowercase "A DB" to "a DB"? Or does the "A" stand for "Atlas" in this case?

Reply:
Atlas DB is correct :) thanks for the catch!